src/algebra/lie_algebra.lean

```diff
@@ -416,6 +421,23 @@ instance lie_subalgebra_lie_algebra (L' : lie_subalgebra R L) :
 local attribute [instance] lie_ring.of_associative_ring
 local attribute [instance] lie_algebra.of_associative_algebra
 
+/-- The embedding of a Lie subalgebra into the ambient space as a Lie morphism. -/
+def lie_subalgebra.subtype
```

Comment:
I know that this `.subtype` name is used elsewhere for the bundled inclusion map, but I really don't like it. I suggest we rename all of them to something like `incl`. Reason: `subtype` doesn't sound like a map at all.
(Of course this has nothing much to do with this PR. But I hope that a little code fairy can fix this "issue" in another PR :wink:)

Reply:
Agreed; I prefer `.incl` too. I've just renamed this instance and made a note to follow up after this PR to fix the other instances (`submodule.subtype`, ...).